lib/mayaUsd/render/vp2ShaderFragments/UsdPreviewSurfaceLightAPI1.xml

```diff
@@ -65,7 +66,7 @@ limitations under the License.
         <connect from="mayaShaderGeom_Float4GetY.output" to="usdPreviewSurfaceLightingAPI1.NdotV" name="NdotV"/>
         <connect from="mayaShaderGeom_Float4GetZ.output" to="usdPreviewSurfaceLightingAPI1.NdotH" name="NdotH"/>
         <connect from="mayaShaderGeom_Float4GetW.output" to="usdPreviewSurfaceLightingAPI1.VdotH" name="VdotH"/>
-        <connect from="mayaShaderGeom_Float4GetW.output" to="usdPreviewSurfaceCombiner.VdotH" name="VdotH"/>
+        <connect from="mayaShaderGeom_Float4GetY.output" to="usdPreviewSurfaceCombiner.VdotH" name="NdotV"/>>
```

Comment:
Fixes issue 1. There is no H vector for environment maps, so Maya was re-using the last H it saw. Use VdotN instead.

Reply:
Yes, indeed a typo in the comment.